website/docs/r/qldb_ledger.html.markdown

```diff
@@ -0,0 +1,53 @@
+---
+layout: "aws"
+page_title: "AWS: aws_qldb_ledger"
+description: |-
+  Provides an QLDB Resource resource.
+---
+
+# Resource: aws_qldb_ledger
+
+Provides an AWS Quantum Ledger Database (QLDB) resource
+
+## Example Usage
+
+```hcl
+resource "aws_qldb_ledger" "sample-ledger" {
+  name = "sample-ledger"
+}
+```
+
+## Example with disabled deletion protection 
+**NOTE:** The deletion protection is enabled by default, so any changes that require a new resource will fail. 
```

Comment:
I think this example can be omitted and the note wording is a little confusing. My suggestion would be to clarify the problem and solution inline:

```suggestion
~> **NOTE:** Deletion protection is enabled by default. To successfully delete this resource via Terraform, `deletion_protection = false` must be applied before attempting deletion. 
```